recipe/meta.yaml

```diff
@@ -38,7 +38,6 @@ requirements:
 test:
   commands:
     - jupyter --help
-    - jupyter nbextension list
```

Comment:
This seems to be its own separate package now, and not something we should include by default in `jupyter` if we are following the PyPI version of the package.

Reply:
https://github.com/conda-forge/jupyter_contrib_nbextensions-feedstock

Reply:
@ocefpaf and @minrk, do you agree that this seems like the right fix?

Reply:
I'm on only to investigate further but looks good to me.

Reply:
@minrk, any feedback? If I don't here soon, I'll go ahead and merge to avoid holding up the Python 3.12 migration.

Reply:
Yes, it's from the `notebook` package pre-7.0. Removing it here is correct.